Source/Core/VideoBackends/D3D12/D3D12.vcxproj

```diff
@@ -41,12 +49,26 @@
       <ForcedIncludeFiles />
     </ClCompile>
   </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|ARM64'">
```

Comment:
These blocks (as well as the ones in D3DCommon) can also be merged with existing x64 blocks.

Kinda related...why are these projects disabling pch?

Reply:
No idea. Seems it's something that VS did at some point. I removed the whole conditional and it seems to work fine...